### PR TITLE
Run.sh file added to help run the model on boot

### PR DESCRIPTION
**Pull request by Josephshark**
Added the run.sh file used to run the main python file
After updating the permissions in terminal with:

cd /home/warg/computer-vision-python
chmod +x run.sh

The to test use ./run.sh to run the python file

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd /home/warg/computer-vision-python
+source venv/bin/activate
+python main_2023.py
```
Comment (Xierumeng) [inline run.sh:4]: Nit: Add empty newline at end of file.

**Comment by Xierumeng** [comment]
Next time, please use a branch in the WARG organization rather than a fork.

**Comment by Josephshark** [comment]
Ok will do.

**Review by Xierumeng — APPROVED** [review]
Change your PR title to something more descriptive, then click the merge button.

**Pull request merged**